packages/server/lib/modes/record.js

```diff
@@ -784,6 +784,8 @@ const createRunAndRecordSpecs = (options = {}) => {
         })
 
         if (response === responseDidFail) {
+          debug('`responseDidFail` equals `response`, allowing browser to hang until it is killed: Response %o', { responseDidFail })
```

Comment:
why not just kill the process?

Reply:
This seems like the safest (easiest) option. I don't have context for why this was setup this way, but now we will have logs when/ if it happens.